apps/learn.svelte.dev/vercel.json

```diff
@@ -0,0 +1,10 @@
+{
+	"$schema": "https://openapi.vercel.sh/vercel.json",
+	"redirects": [
+		{
+			"source": "/(.*)",
```

Comment:
this will be cumbersome; we basically need to go through all the links and find out the right equivalent on the new one